src/ContainerFactory.ts

```diff
@@ -78,6 +81,27 @@ export function createContainer(): Container {
     container.bind(PenaltyIdentifierSchemaFactory)
         .toConstantValue(new PenaltyIdentifierSchemaFactory(getEnvOrThrow('ALLOWED_COMPANY_PREFIXES')));
 
+    const companyAuthConfig: CompanyAuthConfig = {
+        oath_scope_prefix: getEnvOrThrow('OATH_SCOPE_PREFIX'),
+        accountUrl: getEnvOrThrow('ACCOUNT_URL'),
+        accountRequestKey: getEnvOrThrow('OAUTH2_REQUEST_KEY'),
+        accountClientId: getEnvOrThrow('OAUTH2_CLIENT_ID'),
+        chsUrl: getEnvOrThrow('ACCOUNT_WEB_URL'),
+    };
+
+    const sessionStoreForAuthConfig = {
```

Comment:
I suggest creating class for this config which has static method `createFromEnvironmentVariables()` which returns instance populated with values read from env variables. This way it can be used here and in base controller to remove duplicates and keeps defaults consistent.

Example usage:

```
const config = SessionConfig.createFromEnvironmentVariables();
```

Reply:
I have created the class and used it for both the base controller and the container factory